README.md

```diff
@@ -73,7 +73,7 @@ echo $PATH
 sudo curl -o /usr/local/bin/scw -L "https://github.com/scaleway/scaleway-cli/releases/download/v2.0.0-beta.1/scw-v2.0.0-beta.1-darwin-x86_64"
```

Comment:
We don't need `sudo` at all for macOS. At least for standard macOS installations.
Can you remove it in `curl` and `chmod` please? :)

Reply:
Indeed 👍